tracing-subscriber/src/fmt/mod.rs

```diff
@@ -664,6 +742,38 @@ impl Default for Settings {
     }
 }
 
+/// Install a global tracing subscriber that listens for events and
+/// filters based on environment variables, if one is not already set.
```

Comment:
Nit: can this maybe link to the default env variable (https://docs.rs/tracing-subscriber/0.1.5/tracing_subscriber/filter/struct.EnvFilter.html#associatedconstant.DEFAULT_ENV) or name it here? Like 
```suggestion
/// filters based the value of the [RUST_LOG environment variable](../filter/struct.EnvFilter.html#associatedconstant.DEFAULT_ENV), if one is not already set.
```